packages/amplify-category-api/src/provider-utils/awscloudformation/cdk-stack-builder/apigw-stack-builder.ts

```diff
@@ -12,20 +12,23 @@ const CFN_TEMPLATE_FORMAT_VERSION = '2010-09-09';
 const ROOT_CFN_DESCRIPTION = 'API Gateway Resource for AWS Amplify CLI';
 
 export class AmplifyApigwResourceStack extends cdk.Stack implements AmplifyApigwResourceTemplate {
-  _scope: cdk.Construct;
-  restApi!: apigw.CfnRestApi;
+  restApi: apigw.CfnRestApi;
   deploymentResource: apigw.CfnDeployment;
-  _lambdaPermission: lambda.CfnPermission;
-  _props: ApigwInputs;
   paths: $TSObject;
   policies: { [pathName: string]: ApigwPathPolicy };
-  _cfnParameterMap: Map<string, cdk.CfnParameter> = new Map();
+  private _scope: cdk.Construct;
+  private _props: ApigwInputs;
+  private _cfnParameterMap: Map<string, cdk.CfnParameter> = new Map();
+  _cfnParameterValues: $TSObject;
```

Comment:
Is this supposed to be private?

Reply:
Changed it to private, added a getter for transform class to use.